hack/README.md

```diff
@@ -0,0 +1,69 @@
+docker/hack
+============================
+
+## About
+
+`hack` contains a collection of scripts used by Docker. If there are any issues regarding the intention of a particular script
```

Comment:
It could even be something like `contains a collection of script used to build and manage this repository` 👼